src/Menu/CachedBackendMenuBuilder.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Menu;
+
+use Psr\SimpleCache\CacheInterface;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+class CachedBackendMenuBuilder
+{
+    /** @var CacheInterface */
+    private $cache;
+
+    /** @var BackendMenuBuilder */
+    private $menuBuilder;
+
+    /** @var Stopwatch */
+    private $stopwatch;
+
+    public function __construct(CacheInterface $cache, BackendMenuBuilder $menuBuilder, Stopwatch $stopwatch)
+    {
+        $this->cache = $cache;
+        $this->menuBuilder = $menuBuilder;
+        $this->stopwatch = $stopwatch;
+    }
+
+    public function getMenu(bool $pretty = false): string
```

Comment:
what pretty means? how pretty is menu?

Reply:
Right, it's a flawed name.. It's about pretty-printing the JSON.. Updated..